make-a-pacscript.md

```diff
@@ -108,15 +107,15 @@ It's an array of pacscripts dependencies. Should be used when the pacscript need
 
 #### `ppa`
 
-It's an array that you can use to install ppa's. It is highly discouraged to use ppa's as they are fundamentaly broken. You do not need to include `ppa:` in the array as pacstall does that for you and it's easier for the end user. You should only use a ppa if absolutely necessary. Consider making a pacscript instead and putting it in `pacdeps`
+It's an array that you can use to install ppa's. It is highly discouraged to use ppa's as they are fundamentaly broken. Consider making a pacscript instead and putting it in `pacdeps`. Only use one if it's absolutely necessary. You do not need to include `ppa:` in the array as pacstall does that for you and it's easier for the end user.
 
 #### `optdepends`
 
-It's where you put optional dependencies. Pacstall will ask the user if they want to install them after the package finishes installing.
+It's where you put optional dependencies. Pacstall will ask the user if they want to install them before the package starts installing.
 
 #### `breaks` 
 
-This is where you put the packages that would break if you install this package. An example would be foo and foo-git. They both install to the same files, but different sources. It is not needed unless you are making a `-git` package or `-bin`.
+This is where you put the packages that would break if you install this package. An example would be foo and foo-git. They both install to the same files, but different sources. It is not needed unless you are making a `-git`, `-bin` or `-deb` package.
```

Comment:
what about `-app` and building tho?

Reply:
`-app`s are self contained and don’t conflict with the system